pkg/cmd/label/list.go

```diff
@@ -120,3 +148,14 @@ func printLabels(io *iostreams.IOStreams, labels []label) error {
 func listHeader(repoName string, count int, totalCount int) string {
 	return fmt.Sprintf("Showing %d of %s in %s", count, utils.Pluralize(totalCount, "label"), repoName)
 }
+
+func (opts listQueryOptions) OrderBy() map[string]interface{} {
```

Comment:
```suggestion
func (opts listQueryOptions) OrderBy() map[string]string {
```
I don't think there is any reason why we can't specify the exact return value here.